src/convergence_factors.jl

```diff
@@ -0,0 +1,145 @@
+import Dates
+import SciMLBase
+import ClimaComms
+import ClimaCore as CC
+import ClimaTimeSteppers as CTS
+import ClimaCoupler:
+    Checkpointer,
+    FieldExchanger,
+    FluxCalculator,
+    Interfacer,
+    TimeManager,
+    Utilities
+
+function compute_actual_rho(physical_values)
+    sigma_o = sqrt(abs(physical_values[:w_min]) / (2 * physical_values[:alpha_o])) * (1 + im)
+    sigma_a = sqrt(abs(physical_values[:w_min]) / (2 * physical_values[:alpha_a])) * (1 + im)
+    eta_AO = physical_values[:C_AO] * abs(physical_values[:u_atm] - physical_values[:u_oce]) * physical_values[:rho_atm] * physical_values[:c_atm]
+    eta_OI = physical_values[:C_OI] * abs(physical_values[:u_oce]) * physical_values[:rho_oce] * physical_values[:c_oce]
+    eta_AI = physical_values[:C_AI] * abs(physical_values[:u_atm]) * physical_values[:rho_atm] * physical_values[:c_atm]
+    return abs((1 - physical_values[:a_i])^2 * eta_AO^2 / ((physical_values[:k_oce] * sigma_o * (1 / tanh(sigma_o * (physical_values[:h_oce] - physical_values[:z_0numO]))) + (1 - physical_values[:a_i]) * eta_AO + physical_values[:a_i] * eta_OI) * (physical_values[:k_atm] * sigma_a * (1 / tanh(sigma_a * (physical_values[:h_atm] - physical_values[:z_0numA]))) + (1 - physical_values[:a_i]) * eta_AO + physical_values[:a_i] * eta_AI)))
+end
+
+function compute_numerical_conv_fac(physical_values; return_conv_facs=true, plot_conv_facs=false, print_conv_facs=false)
```

Comment:
and here `compute_ρ_numerical` or `compute_ρ_discrete` or `compute_ρ_exp` or similar?